src/components/comment-input.component.js

```diff
@@ -76,8 +77,14 @@ export class CommentInput extends Component {
     };
   }
 
-  handleSubmit = (body: string): void => {
-    this.props.onSubmitEditing(body);
+  handleSubmitEditing = (body: string): void => {
+    if (Platform.OS === 'android') {
```

Comment:
@housseindjirdeh If there is this problem on iOS, can this condition be removed? It is necessary to check on iOS with these changes, and then you can remove this condition, and check without it.